model/transform/src/main/scala/aqua/model/transform/topology/Topology.scala

```diff
@@ -371,7 +372,7 @@ object Topology extends Logging {
           SeqNext
         case (_, _: ForTag) =>
           For
-        case (_, ParTag | ParTag.Detach) =>
+        case (_, ParTag) => // No begin optimization for detach
```

Comment:
`Don't being`?

Reply:
`begin` optimization is disabled for `detach`. The comment I left will help me more. Don't understand your reply.

Reply:
I couldn't read it, so I guessed there's a typo. From what you've explained, it seems I guessed correctly.

Reply:
```
begin optimization is disabled for detach
```
sounds very easy to understand :)